src/main/java/uk/gov/hmcts/darts/common/repository/CaseRepository.java

```diff
@@ -23,11 +22,11 @@ Optional<CourtCaseEntity> findByCaseNumberAndCourthouse(String caseNumber,
                                                             CourthouseEntity courthouse);
 
     @Query("""
-        SELECT case.caseNumber
-        FROM CourtCaseEntity case
-        WHERE case.closed = false
-        and case.caseNumber in :caseNumbers
-        and case.courthouse.courthouseName = upper(:courthouseName)
+        SELECT c.caseNumber
+        FROM CourtCaseEntity c
+        WHERE c.closed = false
+        and c.caseNumber in :caseNumbers
+        and c.courthouse.courthouseName = upper(:courthouseName)
```

Comment:
It's a general gripe I have with how most of our JPQL is already written, but out of interest, what's the benefit of changing to single char var names here versus a more verbose name? I think the extra verbosity of `case` helps the readability, especially when we're dealing with other `c`-things like courthouses etc.

I've noticed that IntelliJ complains with the alias `case` (is it a reserved JPQL keyword maybe?) But maybe we could use `courtCase` instead.

What do you think?

Reply:
This was marked as resolved but seems like it's still there as `c`